### PR TITLE
update args_test.go unittest

### PR DESCRIPTION
**Pull request by yxxhero**
Signed-off-by: yxxhero <aiopsclub@163.com>

update args_test.go unittest

1. Optimize unit testing of GetArgs
2. add unittest for SetArgs

### CODE REVIEW

**Comment by mumoshu** [comment]
@yxxhero Would you mind writing a line or two of the description that covers the intention or the goal of a pull request?! It doesn't need to "correct" English as I'm not a native English speaker/writer and I definite'y don't believe in my English skill myself.

I can't thank you more for your contributions to the project, but it's always a guess-game for me without any description, which might result in me taking more time to review than how long it might took to author the change and the pull request myself 🙏

File: pkg/argparser/args.go [hunk 1]
```diff
@@ -18,7 +18,12 @@ type argMap struct {
 	flags []string
 }
 
+// SetArg sets a flag and value in the map
 func (a *argMap) SetArg(flag, arg string, isSpace bool) {
+	// if flag is empty, return
+	if len(flag) == 0 {
+		return
+	}
```
Comment (mumoshu) [inline pkg/argparser/args.go:26]: Is this new if block needed? Is this tested in the updated test code? 🤔

**Comment by yxxhero** [comment]
> @yxxhero Would you mind writing a line or two of the description that covers the intention or the goal of a pull request?!(Like you tried to reduce the repetition in the test code and added a new test function to increase code coverage? It would also be helpful to say if it's totally a refactoring, or you added/removed cases, or changed impl)
> 
> It doesn't need to "correct" English as I'm not a native English speaker/writer and I definite'y don't believe in my English skill myself.
> 
> I can't thank you more for your contributions to the project, but it's always a guess-game for me without any description, which might result in me taking more time to review than how long it might took to author the change and the pull request myself 🙏

ok， I will add more describetion for PR. Thanks very much.

File: pkg/argparser/args.go (hunk 1 above)
Comment (yxxhero) [reply pkg/argparser/args.go:26]: when flag is empty. It makes no sense. Added in unit test.

**Review by itscaro — APPROVED** [review]

**Review by mumoshu — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**